media/README.md

```diff
@@ -1,20 +1,29 @@
-# Screen component
+# Media component
 
 ## Installing
 This project is built with Electron.
 
-Run ```npm install``` from the screen folder to install all dependencies.
+Run ```npm install``` from the media folder to install all dependencies.
 
 ## Running
-Run the project with ```node_modules/electron/dist/electron.exe .``` from the screen folder (use backslashes if running in Windows CMD).
+Run the project with ```node_modules/electron/dist/electron.exe .``` from the media folder (use backslashes if running in Windows CMD).
+
+### Available environment variables
+| Name of variable            | Function                                                                                       | Default value             |
+| --------------------------- | ---------------------------------------------------------------------------------------------- | ------------------------- |
+| SCREENCRASH_COMPONENT_ID    | Set name of component to be displayed in UI and logs                                           | Random string of 16 chars |
+| SCREENCRASH_CORE            | Set address to core                                                                            | localhost:3000            |
+| SCREENCRASH_SUPPORTED_TYPES | Space separated string of what types should be active. Available are: audio, video, image, web | All available types       |
```

Comment:
Could the default take into account SCREENCRASH_NO_WINDOW and SCREENCRASH_NO_AUDIO, and e.g. remove 'audio' if SCREENCRASH_NO_AUDIO is 'true'? Then we wouldn't need to set this in the normal case (but it's good to have the option).